src/components/map/Map.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import L from 'leaflet';
+import { Draw, Control } from 'leaflet-draw';
+
+const MAP_MIN_ZOOM = 2;
+const MAP_INITIAL_ZOOM = 3;
+const BASEMAP_TILE = 'http://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}';
+const MAP_CENTER = [51.505, -0.09];
+const BASEMAP_ATTRIBUTION = 'Tiles &copy; Esri &mdash; Source: Esri, i-cubed, USDA, USGS, AEX, GeoEye, Getmapping, Aerogrid, IGN, IGP, UPR-EGP, and the GIS User Community';
+
+const activeLayers = new L.FeatureGroup();
+const drawControlFull = {
+  position: 'topright',
+  draw: {
+      polyline: false,
+      polygon: {
+          allowIntersection: false, // Restricts shapes to simple polygons
+          drawError: {
+              color: '#e1e100', // Color the shape will turn when intersects
+              message: '<strong>Oh snap!<strong> you can\'t draw that!' // Message that will show when intersect
+          },
+          shapeOptions: {
+              color: '#bada55'
+          }
+      },
+      circle: false,
+      rectangle: {
+          shapeOptions: {
+              clickable: false
+          }
+      },
+      marker: false
+  },
+  edit: {
+      featureGroup: activeLayers, //REQUIRED
+      remove: true
+  }
+};
+const drawControlEdit = {
+  position: 'topright',
+  draw: {
+      polyline: false,
+      polygon: false,
+      circle: false,
+      rectangle: false,
+      marker: false
+  },
+  edit: {
+      featureGroup: activeLayers, //REQUIRED
+      remove: true
+  }
+};
+
```

Comment:
I have placed these in the constants file, but I am not certain these shouldn't live inside the component for easy understanding, as they are very specific and have a more complex structure.